### DISCUSSION TITLE
Did the new Blitz.js version 0.38.0 break onSubmit?

### DISCUSSION THREAD

**Discussion by skorotkiewicz** [blitz-js/blitz#2509] [Q&A] [answered]
Yesterday I upgraded Blitz.js from version 0.36.4 to 0.38.0, today I wanted to start a new project, I created "Statuses (1)" I wanted to add a test Status, but unfortunately onSubmit does not work, it does not run the code in onSubmit at all.
In the old version 0.36 it worked smoothly.

I use on the old version and on the newer version: `React Final Form`

```bash
$ blitz generate all status status:string (1)
```

```javascript
      <h1>Create New Status</h1>

      <StatusForm
        submitText="Create Status"
        onSubmit={async (values) => {
          console.log(values) // <-- nothing, onSubmit don't starts when trying to add a new status

          try {
            const status = await createStatusMutation(values)
            router.push(`/statuses/${status.id}`)
          } catch (error) {
            console.error(error)
            return {
              [FORM_ERROR]: error.toString(),
            }
          }
        }}
      />

```

**Comment by flybayer** [comment, marked as answer]
Yes 😏 See the workaround and fix in https://github.com/blitz-js/blitz/pull/2511

**Reply by skorotkiewicz** [reply]
Thanks, it works now :)
It's a beta, so we need to forgive little bugs ;)
